examples/acme.rs

```diff
@@ -0,0 +1,85 @@
+extern crate hornet; 
+extern crate rand;
+
+use hornet::client::Client;
+use hornet::client::metric::*;
+use rand::random;
+use std::thread;
+use std::time::Duration;
+
+fn main() {
+
+    let products = ["Anvils", "Rockets", "Giant_Rubber_Bands"];
+    let indom = Indom::new(
+        &products,
+        "Acme products",
+        "Most popular products produced by the Acme Corporation"
+    );
+    
+    /* create three instance metrics */
+
+    let mut counts = InstanceMetric::new(
+        &indom,
+        "products.count",
+        0,
+        Semantics::Counter,
+        Unit::new().count(Count::One, 1).unwrap(),
+        "Acme factory product throughput",
+        "Monotonic increasing counter of products produced in the Acme Corporation\nfactory since starting the Acme production application. Quality guaranteed."
+    ).unwrap();
+
+    let sec_unit = Unit::new().time(Time::Sec, 1).unwrap();
+
+    let mut times = InstanceMetric::new(
+        &indom,
+        "products.time",
+        0,
+        Semantics::Counter,
+        sec_unit,
+        "Machine time spent producing Acme products",
+        "Machine time spent producing Acme Corporation products. Does not include\ntime in queues waiting for production machinery."
+    ).unwrap();
+
+    let mut queue_times = InstanceMetric::new(
+        &indom,
+        "products.queuetime",
+        0,
+        Semantics::Counter,
+        sec_unit,
+        "Queued time while producing Acme products",
+        "Time spent in the queue waiting to build Acme Corporation products,\nwhile some other Acme product was being built instead of this one."
+    ).unwrap();
+
+    /* create a client, register the metrics with it, and export them */
+
+    Client::new("acme").unwrap()
+        .begin_all(1, 3, 3, 0).unwrap()
```

Comment:
I agree. The solution would be to have a function that registers all metrics at once. I plan to do that after we finish the higher-level metrics like Counter, Histogram, etc.